src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/GridLayouter.cs

```diff
@@ -46,6 +46,29 @@ public class GridLayouter : ItemsLayouter
         private Timer requestLayoutTimer = null;
         private bool isSourceEmpty;
 
+        /// <inheritdoc/>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected new IGroupableItemSource Source => collectionView?.InternalSource;
+
+        /// <summary>
+        /// Span Size
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected int SpanSize => spanSize;
+
+        /// <summary>
+        /// Size Candidate
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected (float Width, float Height) SizeCandidate => sizeCandidate;
+
+        /// <summary>
+        /// Visible ViewItem.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected new List<GroupInfo> GroupItems => groups;
```

Comment:
How about declaring `ItemLayouter.GroupItems` as a virtual property? And then `GridLayouter.GroupItems` can `override` it.

Reply:
sure. it makes sense. thank you.